### DISCUSSION TITLE
After using --profile-directory, the chrome of the specified user is correctly opened, but the webpage cannot continue to be opened

### DISCUSSION THREAD

**Discussion by woshixiongda** [ultrafunkamsterdam/undetected-chromedriver#1346] [General]
from capmonstercloudclient import CapMonsterClient, ClientOptions
from capmonstercloudclient.requests import RecaptchaV2ProxylessRequest
import asyncio
from datetime import datetime
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.common.by import By
import random
import string
import random
import time
from selenium.webdriver.common.action_chains import ActionChains
import seleniumwire.undetected_chromedriver as uc

# 代理IP文本文件路径
proxies_file_path = r"C:\Users\USER\Desktop\proxies.txt"

# 配置 Capmonster
client_options = ClientOptions(api_key="KEY")
cap_monster_client = CapMonsterClient(options=client_options)

async def solve_captcha():
    recaptcha2request = RecaptchaV2ProxylessRequest(
        websiteUrl="SITE",
        websiteKey="SITE KEY"
    )
    return await cap_monster_client.solve_captcha(recaptcha2request)

# 打开代理IP文本文件
with open(proxies_file_path, 'r') as file:
    proxies = file.read().splitlines()

while True:
    # 随机选择一个代理IP
    selected_proxy = random.choice(proxies)

    # 提取代理IP的相关信息
    proxy_parts = selected_proxy.split(':')
    ip = proxy_parts[0]
    port = proxy_parts[1]
    username = proxy_parts[2]
    password = proxy_parts[3]

    # 配置代理选项
    proxy_options = {
        'proxy': {
            'http': f'http://{username}:{password}@{ip}:{port}',
            'https': f'https://{username}:{password}@{ip}:{port}',
        }
    }

    # 配置Chrome
    chrome_options = uc.ChromeOptions()
    chrome_options.add_argument(r'--user-data-dir=C:\Users\XIONGDA\AppData\Local\Google\Chrome\User Data\\')
    chrome_options.add_argument(r'--profile-directory=Profile 2')                    
    driver = uc.Chrome(seleniumwire_options=proxy_options, options=chrome_options)

    # 打开目标网页
    driver.get("URL")

**Comment by woshixiongda** [comment]
Solved after restarting the computer

**Comment by bullbrownrice** [comment]
+1. Solved after restarting
